utils/storage/getSecureStorageItem.ts

```diff
@@ -0,0 +1,23 @@
+import * as SecureStore from 'expo-secure-store';
+
+const getSecureStorageItem = async (keyIdentifier: string): Promise<string> => {
+	try {
+		const keychainService = `secretKey-${keyIdentifier}`;
+		const secretKey = await SecureStore.getItemAsync(keyIdentifier, {
+			keychainService,
+		});
+
+		if (secretKey == null) {
```

Comment:
```suggestion
		if (!secretKey) {
```

Otherwise here we should use strict javascript compare `===`